packages/devtools/test/fixtures/color_console_output_app.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:developer' as developer;
```

Comment:
[nit] Blank line below this.

https://dart.dev/guides/language/effective-dart/style#do-place-dart-imports-before-other-imports

Reply:
done